backend/unit/unit_test.go

```diff
@@ -0,0 +1,207 @@
+package unit
+
+import (
+	"testing"
+	"time"
+
+	"github.com/short-d/app/mdtest"
+)
+
+type fakeDuration int64
+
+func TestParseDuration(t *testing.T) {
+	t.Parallel()
```

Comment:
Let's add empty string and multiple digits case for each unit.

Reply:
Empty string is handled the same way for each unit so it's not an added value for tests, however we can add test when there units but no value i.e. "m", "w"

Reply:
@arberiii When writing unit tests, we should know as less internal implementation as possible. What happens if we changed the code. Then the test won't fail. The goal of the test is to make sure we are safe from refactors.

Reply:
Rethinking about it, that is a really valuable lesson. Thank you!